recipe-server/normandy/recipes/tests/test_commands.py

```diff
@@ -158,3 +176,35 @@ def test_it_resigns_signed_recipes_with_force(self, mocked_autograph):
         call_command('update_recipe_signatures', '--force')
         r.refresh_from_db()
         assert r.signature.signature is not 'old signature'
+
+
+@pytest.mark.django_db
+class TestUpdateActionSignatures(object):
```

Comment:
We could also use a test that it does not sign signatures that haven't expired yet.